policies-procedures-and-docs/payments/honorarium.md

```diff
@@ -0,0 +1,38 @@
+# OLS Honoraria Policy
```

Comment:
Should be moved to the Open Seeds folder

Reply:
Moved. Thank you. 🙏